care/facility/api/viewsets/asset.py

```diff
@@ -128,6 +130,17 @@ def retrieve(self, request, *args, **kwargs):
         return Response(hit)
 
 
+class AssetAvailabilityFilter(filters.FilterSet):
+    external_id = filters.CharFilter(field_name="asset__external_id")
+
+
+class AssetAvailabilityViewSet(ListModelMixin, RetrieveModelMixin, GenericViewSet):
+    queryset = AssetAvailabilityRecord.objects.all().select_related("asset")
```

Comment:
Are we sure we want to open this endpoint to everyone? cc @gigincg

Reply:
@vigneshhari Gigin said it's okay for this endpoint to be public for now.